src/benchmark/scala/io/iohk/ethereum/bench/Benchmark.scala

```diff
@@ -0,0 +1,97 @@
+package io.iohk.ethereum.bench
+
+//import io.iohk.ethereum.domain.BlockchainImpl
+import java.io.File
+import java.util.concurrent.Executors
+
+import akka.actor.ActorSystem
+import io.iohk.ethereum.consensus.{ConsensusConfigBuilder, StdConsensusBuilder}
+import io.iohk.ethereum.domain.Block
+import io.iohk.ethereum.domain.Block._
+import io.iohk.ethereum.ledger.{BlockImportedToTop, DuplicateBlock}
+import io.iohk.ethereum.nodebuilder._
+import io.iohk.ethereum.utils.{Config, Logger}
+import org.bouncycastle.util.encoders.Hex
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.io.Source
+/*
+* Simple program to benchmark block import performance.
+* It requires file with rlp encoded blocks to be located in main mantis dir
+* All processed data will be saved to dir specified in rocksdb config specified in main Application.conf
+* Of cource block import is not whole story of mantis performance but it is great part of it.
+*
+*
+* */
+//scalastyle:off
+object Benchmark extends App with GenesisDataLoaderBuilder
+  with Logger
+  with ActorSystemBuilder
+  with StorageBuilder
+  with BlockchainConfigBuilder
+  with BlockchainBuilder
+  with VmConfigBuilder
+  with VmBuilder
+  with SyncConfigBuilder
+  with ShutdownHookBuilder
+  with ConsensusConfigBuilder
+  with StdConsensusBuilder
+  with StdLedgerBuilder {
+
+  override implicit lazy val system = ActorSystem("Test_System")
+
+  prepareDbDir(Config.Db.RocksDb.path)
+
+  val exe = Executors.newFixedThreadPool(4)
+
+  val context = ExecutionContext.fromExecutor(exe)
+
+  val source = Source.fromFile("blocks.txt")
+
+  genesisDataLoader.loadGenesisData()
+
+  val lines = source.getLines()
+  val startTime = System.nanoTime()
+  lines.foreach { line =>
+    val block = Hex.decode(line).toBlock
```

Comment:
I think that `Future.traverse` would be a good fit here: https://www.scala-lang.org/api/current/scala/concurrent/Future$.html#traverse[A,B,M[X]%3C:TraversableOnce[X]](in:M[A])(fn:A=%3Escala.concurrent.Future[B])(implicitcbf:scala.collection.generic.CanBuildFrom[M[A],B,M[B]],implicitexecutor:scala.concurrent.ExecutionContext):scala.concurrent.Future[M[B]]

Also there's one thing that concerns me a bit - In regular sync imports are called from Akka actor which can also impact its performance to some degree (implicit switches of threads between messages). I don't know if this is something that's relevant in context of this benchmark though. Maybe it would be a good way for future to set up `Importer` actor here with fake fetcher, etc.?